nodeadm/internal/api/status.go

```diff
@@ -3,6 +3,7 @@ package api
 import (
 	"context"
 	"fmt"
+	featuregates "github.com/awslabs/amazon-eks-ami/nodeadm/internal/feature-gates"
```

Comment:
if you name the directory `featuregates` or just `features` instead of `feature-gates` then you don't need to rewrite the import every time 😉